fontc_crater/src/args.rs

```diff
@@ -41,6 +41,9 @@ pub(super) struct CiArgs {
     /// This should be consistent between runs.
     #[arg(short = 'o', long = "out")]
     pub(super) out_dir: PathBuf,
+    /// skip gftools mode (to reduce target count when running locally)
+    #[arg(long)]
+    pub(super) skip_gftools: bool,
```

Comment:
nit: google-wide guidance is avoid negative flags. Applied here I would mak the flag just `gftools: bool` and default it to true